### PR TITLE
Add support for GitLab services as 'systemd' units

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 
+sudo: True
```
Comment (bfabio) [inline .travis.yml]: This should read `become: True`

Comment (drybjed) [inline .travis.yml]: No, this is for Travis-CI, not Ansible. :-)

Comment (bfabio) [inline .travis.yml]: Oops, sorry. :)

**Comment by cultcom** [comment]
your changes look good to me and I just learned something new with the "slices" ;-)

**Pull request merged**